### PR TITLE
For Ember Times No. 152: Writeup on accessibility of your existing Ember app blog post

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2020-06-12-the-ember-times-issue-152.md [hunk 1]
```diff
@@ -36,14 +37,13 @@ READMORE
...
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Using Ember you might be familiar with a set of tools that you can already use today to [make your app accessible](https://guides.emberjs.com/release/accessibility/). But what do you do if you are working on an already **mature Ember app** that needs a lot of improvements to **become accessible**? Where do you even start?
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In her recent blog post, [Samanta de Barros (@sdebarros)](https://github.com/sdebarros) shares [her findings](https://simplabs.com/blog/2020/06/02/how-to-improve-the-accessibility-of-your-app/) on improving the accessibility of an Ember app by employing an automated, gradual and **a11y-focussed testing strategy**, semantic markup and screen reader testing.
+
+If you want to find out how to level up your app's accessibility, read [her blog post on the simplabs blog](https://simplabs.com/blog/2020/06/02/how-to-improve-the-accessibility-of-your-app/) and if you learned something form it, be sure to share it with your Ember friends, too!
```
Comment (ijlee2) [inline source/2020-06-12-the-ember-times-issue-152.md]:
```suggestion
If you want to find out how to level up your app's accessibility, read [her blog post on the simplabs blog](https://simplabs.com/blog/2020/06/02/how-to-improve-the-accessibility-of-your-app/) and if you learned something from it, be sure to share it with your Ember friends, too!
```

**Review by ijlee2 — APPROVED** [review]
Looks great, I enjoyed reading Samanta's article and her encouraging more teams to start testing how accessible their apps are. Thank you for the write-up!

File: source/2020-06-12-the-ember-times-issue-152.md [hunk 2]
```diff
@@ -36,14 +37,13 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Learn how to make your existing Ember app accessible 💪](https://simplabs.com/blog/2020/06/02/how-to-improve-the-accessibility-of-your-app/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Using Ember you might be familiar with a set of tools that you can already use today to [make your app accessible](https://guides.emberjs.com/release/accessibility/). But what do you do if you are working on an already **mature Ember app** that needs a lot of improvements to **become accessible**? Where do you even start?
```
Comment (abhilashlr) [inline source/2020-06-12-the-ember-times-issue-152.md]:
```suggestion
Using Ember, you might be familiar with a set of tools that you can already use today to [make your app accessible](https://guides.emberjs.com/release/accessibility/). But what do you do if you are working on an already **mature Ember app** that needs a lot of improvements to **become accessible**? Where do you even start?
```

Should there be a `,`?

**Review by abhilashlr — APPROVED** [review]
The writeup looks great @jayjayjpg 🥇

**Comment by jayjayjpg** [comment]
Thank you for all the suggestions @abhilashlr & @ijlee2 :bowing_woman:

**Pull request merged**